lib/fortnox/api/models/base.rb

```diff
@@ -19,8 +19,12 @@ def self.attribute( name, *args )
         end
 
         def self.new( hash )
-          obj = preserve_meta_properties( hash ) do
-            super( hash )
+          begin
+            obj = preserve_meta_properties( hash ) do
+              super( hash )
+            end
+          rescue Dry::Struct::Error, Dry::Types::ConstraintError, Dry::Types::MissingKeyError => e
```

Comment:
If I implement the above, I get some errors which I think is related to the other failing specs. I'll push what I have and let you have a look at that too @d-Pixie :)